database/test/conftest.py

```diff
@@ -733,3 +827,37 @@ def lifecycle_date_instance(session, code_instance):
     instance = models.LifeCycleDate(lifecycle_status=code_instance)
     session.add(instance)
     return instance
+
+
+@pytest.fixture(scope="module")
+def complete_test_plan(
+    session: Session,
+    plan_instance: models.Plan,
+    land_use_area_instance: models.LandUseArea,
+    plan_regulation_group_instance: models.PlanRegulationGroup,
```

Comment:
@msorvoja Had to recheck, but I think they still need to be included as fixtures.

They are not *modified* in any way, but the fixtures have to be created so that the created test plan will have land use areas and plan regulation groups. Otherwise they will not exist in the database, so the whole plan will fail. They *might* be created by other fixtures, if we are using other fixtures that explicitly refer to these fixtures, but I don't want to go through all the fixtures to make sure that one of them will create these objects in the database.

In my opinion, it is better to explicitly state here that these fixtures will have to be created, otherwise the tests will fail.